Dockerfile

```diff
@@ -4,29 +4,20 @@
 ##
 #
 
-# Let's start with a fresh debian jessie
-FROM debian:jessie
+FROM debian:latest
 
-# Some generic information
-MAINTAINER Charles-Antoine Mathieu
-MAINTAINER Mathieu Bodjikian
+ADD plikd.cfg /home/plik/server/plikd.cfg
 
-RUN apt-get update && apt-get install -y ca-certificates
+RUN apt update && apt install -y ca-certificates curl && \
+        useradd plik && \
+        curl -s https://api.github.com/repos/root-gg/plik/releases/latest | grep -o "https://.*linux-64bits.tar.gz" | xargs curl -L --output /tmp/plik.tar.gz && \
```

Comment:
we can't use release url as plik source because this container (afaik) is use to test some dev versions

Reply:
The missing piece here seems to be the Makefile. The way the Dockerfile is currently build the Makefile generates the go binaries and then builds the image. This would break with this change.

Maybe if the use of make is too complicated for some users (you need to have make and golang installed after all) a dedicated dockerfile could be handy? (like Dockerfile.dev or Dockerfile.release)

Reply:
Hello,
thank you for your feedback so far.

I went a step further and included the build environment in the Dockerfile and used a multistage build to generate a minimalistic image using Golang 1.12.3 in the build time and Alpine 3.9 for the runtime.